src/net_processing.cpp

```diff
@@ -284,8 +314,9 @@ struct Peer {
     /** Work queue of items requested by this peer **/
     std::deque<CInv> m_getdata_requests GUARDED_BY(m_getdata_requests_mutex);
 
-    explicit Peer(NodeId id)
+    Peer(NodeId id, bool blocks_only)
```

Comment:
eeb0e84fc5c259dfd00f7673ad4a51582b100282: Would be nice to rename this from `blocks_only` to `initialize_tx_relay` (or similar). `blocks_only` is confusingly similar named to the setting `-blocksonly`, but it really means block-relay-only.

Moreover, the details *why* tx relay isn't initialized here don't matter *here*, so a bool with a more general name seems fine.

Reply:
Done